src/sonic-config-engine/minigraph.py

```diff
@@ -473,7 +474,18 @@ def parse_dpg(dpg, hname):
                 except:
                     print("Warning: Ignoring Control Plane ACL %s without type" % aclname, file=sys.stderr)
 
-        return intfs, lo_intfs, mvrf, mgmt_intf, vlans, vlan_members, pcs, pc_members, acls, vni
+        mg_tunnels = child.find(str(QName(ns, "TunnelInterfaces")))
+        if mg_tunnels is not None:
+            for mg_tunnel in mg_tunnels.findall(str(QName(ns, "TunnelInterface"))):
+                tunnel_num = "".join([x for x in mg_tunnel.attrib["Name"] if x.isdigit()])
```

Comment:
Why are we getting the tunnel_num seperate? The name should be same as what Graph has.